### DISCUSSION TITLE
群晖NAS部署，反向代理之后出现静态资源找不到解决方法

### DISCUSSION THREAD

**Discussion by BlackWhite2000** [icret/EasyImages2.0#127] [教程]
我使用群晖的Web Station部署好后，再套上域名以及使用反向代理来隐藏端口。
之后出现了静态资源加载错误的问题，看报错是指向错误，静态资源都指向到了一个本地的ip **127.0.0.1**

通过看源代码发现是函数 static_cdn 造成的问题
![image](https://github.com/icret/EasyImages2.0/assets/39222378/4b828d77-23b0-4aef-9d01-111514988537)

由于我是个人使用所以解决方法也比较简单粗暴，我直接注释了函数内的代码
函数代码在 app->function.php
![image](https://github.com/icret/EasyImages2.0/assets/39222378/d52544ea-d4f6-476e-afa2-dcbd27a3b28b)

该问题得以解决

**Comment by BlackWhite2000** [comment]
根据变量的名称意思可能是可以自定义cdn路径的功能。
但是由于我需求很简单只需要一个简易的图床就没管。
很感谢作者提供了如此适合我需求的图床~

**Comment by icret** [comment]
👌
